pkg/elemental/pages/index.vue

```diff
@@ -101,36 +186,162 @@ export default {
 <template>
   <div>
     <Loading v-if="$fetchState.pending" />
+    <!-- Elemental Operator not installed -->
+    <div
+      v-else-if="!isElementalOpInstalled"
+      class="not-installed p-10"
+    >
+      <div class="logo mt-20 mb-10">
+        <img
+          src="../icon.svg"
+          height="64"
+        />
+      </div>
+      <h1 class="mb-20">
+        {{ t("product.elemental") }}
+      </h1>
+      <p
+        class="description"
+        v-html="t('product.description', {}, true)"
```

Comment:
This follows the exact same styling as we have on Kubewarden for it's "welcome" screen when it doesn't have all of the things installed. Neil wanted for us in the future to create a component from these two so that all extension "welcome" screens would follow the same styling.